### DISCUSSION TITLE
If I'm following someone on the fediverse with the friends plugin, do their posts end up in my database?

### DISCUSSION THREAD

**Discussion by djotaku** [akirk/friends#337] [Q&A]
Basically, if I follow high post volume users, am I at risk of using up all the hard drive space where my database is hosted?

**Comment by akirk** [comment]
Yes, the posts end up in your database but you have control over how many posts it will keep. Both in total:
![Screenshot 2024-07-18 at 05 43 29](https://github.com/user-attachments/assets/73475e1a-c37c-49ae-99e3-c68edff07017)
and per user:
![Screenshot 2024-07-18 at 05 44 14](https://github.com/user-attachments/assets/7c5a6eae-3143-4edb-ab8e-37a689edd09f)
As you can see, I personally have my retention settings off and do follow high volume users but it's only text so it's not that much overall.
